pyke/lightcurve.py

```diff
@@ -501,19 +515,126 @@ def plot(self, plottype=None, **kwargs):
             kwargs['color'] = 'C{}'.format(idx)
             lc.plot(label=pl, **kwargs)
 
+class SFFDetrender(Detrender):
+    """Implements a similar procedure as described by Vanderburg and Johnson
+    (2014). Briefly, the algorithm implemented in this class can be described
+    as follows
+
+       (1) Rotate the centroid measurements onto the subspace spanned by the
+           eigenvectors of the centroid covariance matrix
+       (2) Fit a polynomial to the rotated centroids
+       (3) Compute the arclength of such polynomial
+       (4) Fit a BSpline of the raw flux as a function of time
+       (5) Normalize the raw flux by the fitted BSpline computed in step (4)
+       (6) Bin and interpolate the normalized flux as function of the arclength
+       (7) Divide the raw flux by the piecewise linear interpolation done in step [(6)
+       (8) Set raw flux as the flux computed in step (7) and repeat
 
-class Detrender(object):
-    """
+    Attributes
+    ----------
+    poly_order : int
```

Comment:
We use `polyorder` instead of `poly_order` in `flatten()`  and `savgol_filter()`.  Let's change this to `polyorder` for consistency!